docs/articles/documentation/test-api/typescript-support.md

```diff
@@ -16,22 +16,24 @@ TestCafe bundles the TypeScript declaration file with the npm package, so you do
 
 ## Writing Tests with TypeScript
 
+To start writing tests with TypeScript install TestCafe into your project directory. For more information, see [Installing TestCafe](../using-testcafe/installing-testcafe.html#locally).
```

Comment:
To start writing tests with TypeScript, install TestCafe into your project directory. For more information, see [Installing TestCafe](../using-testcafe/installing-testcafe.html#locally)